src/main/java/com/gmail/val59000mc/configuration/MainConfiguration.java

```diff
@@ -999,4 +1018,31 @@ public void setTheEndUuid(String theEndUuid) {
 		this.theEndUuid = theEndUuid;
 	}
 
+
+	/********************/
```

Comment:
Whats the `/**********************/` for? If it's a division line please put some info in what its dividing.